src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -381,17 +381,19 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
         }
 
         [Fact]
-        [OuterLoop]
-        public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
+        [OuterLoop("One second delay in getting server's response")]
+        public async Task Timeout_SetTo30AndGetResponseFromLoopbackQuickly_Success()
         {
-            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
-            const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
-            
-            using (var client = new HttpClient())
+            using (var client = new HttpClient() { Timeout = TimeSpan.FromSeconds(30) })
             {
-                client.Timeout = TimeSpan.FromMinutes(5);
-                var response = client.GetAsync(SlowServer).GetAwaiter().GetResult();
-                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                await LoopbackServer.CreateServerAsync(async (server, url) =>
+                {
+                    Task get = client.GetStringAsync(url);
```

Comment:
nit: I'd prefer we don't use reserved words like `get` which have special meaning in C#.  Can't we call this local variable something like getTask?